spec/order_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'order'
+
+describe Order do
+    let(:menu_klass) {double :menu_klass}
+    let(:text_klass) {double :text_klass, new: text}
+    subject(:order) {described_class.new}
+    let(:text) {double :text}
+
+    let(:send_message) {double :send_message}
+    
+    describe '#initialize' do
+        
+        it 'should initialize with a blank hash of chosen items' do
+            expect(order.chosen_items).to be_empty
+        end
+        
+    end
+    
+    context 'order_selection' do
+        
+        it 'should add items and quantities to the chosen items hash' do
+            order_hash = {:Coke => 5}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)